cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/GoalOptimizer.java

```diff
@@ -135,6 +142,13 @@ public GoalOptimizer(KafkaCruiseControlConfig config,
                                                                  overrideConfigs);
   }
 
+  /**
+   * @return if we have unfixable proposal optimization
```

Comment:
nit: return true if we have unfixable proposal optimzation.

But I think we probably don't need this method anyways.